### DISCUSSION TITLE
Website doesnt show up.

### DISCUSSION THREAD

**Discussion by kittendevv** [is-a-dev/register#10030] [Support] [answered]
![Screenshot 2023-12-12 18 20 58](https://github.com/is-a-dev/register/assets/121097932/15ed4fc0-cd7e-4004-bd01-f0d2c5250e8b)
I fully set up my files on the net2ftp, but when i go to my site it doesnt work.

**Comment by huysam11** [comment]
follow

**Reply by kittendevv** [reply]
What do i need to follow

**Comment by andrewstech** [comment, marked as answer]
> ![Screenshot 2023-12-12 18 20 58](https://github.com/is-a-dev/register/assets/121097932/15ed4fc0-cd7e-4004-bd01-f0d2c5250e8b)
> I fully set up my files on the net2ftp, but when i go to my site it doesnt work.

Fixed

**Reply by kittendevv** [reply]
Super thx
